_usage/migration.md

```diff
@@ -6,73 +6,69 @@ nav_order: 3
+The first step is to run an assessment with Babelfish Compass.
+Run the Babelfish Compass tool on the DDL and determine to what extent the T-SQL code will
+be supported by Babelfish, and identify T-SQL code that may require changing before
+executing against Babelfish.
 
- The first step is to move your existing SQL Server schema to the new
- cluster. Export the DDL (data definition language) of the SQL Server
- database. The DDL is SQL code that describes database objects such
- as those that contain user data (for example, tables, indexes, and
- views) and user-written database code (stored procedures,
- user-defined functions, or triggers).
+To download Babelfish Compass, see here https://github.com/babelfish-for-postgresql/babelfish_compass/releases/latest and here https://github.com/babelfish-for-postgresql/babelfish_compass/blob/main/BabelfishCompass_UserGuide.pdf (for the user guide).
 
-You can use SQL Studio Management Studio (SSMS) to export the DDL.
-Generate the script by clicking on your database in MS SQL Studio and use the
+### Reverse-engineer the SQL Server schema
+
+Once you have run an assessment, migrate your existing SQL Server schema to the new
+cluster. Generate the DDL for all SQL Server user databases which you want to migrate.
+The DDL is SQL code that describes database objects such
+as those that contain user data (for example, tables, indexes, and
+views) and user-written database code (stored procedures,
+user-defined functions, or triggers).
+
+You can use SQL Server Management Studio (SSMS) to reverse-engineer the DDL.
+Generate the script by clicking on your database in SSMS and use the
 functionality provided by the tool to generate the export.
 
 Do not forget to enable triggers, logins, owners, and
 permissions. These are disabled by default in SSMS.
 
-
-### Assess the situation using Compass
-
-Babelfish comes with an assessment tool called Babelfish Compass to help people
-with the migration ahead. Simply run the Compass tool on the DDL and determine
-to what extent the T-SQL code will be supported by Babelfish, and identify T-SQL
-code that may require changing before executing against Babelfish. 
-
- 
 ### Deploying the schema and migrating the data
 
-Run the DDL on your new Babelfish server to recreate your schema on Babelfish. 
-Make SQL code adjustments as needed.
-
-In case you are facing errors or problems, we recommend you check out our [discussion about
-differences in functionality](/docs/usage/limitations-of-babelfish).
+Make SQL code adjustments as needed. In case you are facing errors or problems,
+we recommend you check out our [discussion about differences in functionality](/docs/usage/limitations-of-babelfish).
+Run the DDL on your new Babelfish server to recreate your schema on Babelfish,
+either with sqlcmd or with SSMS. 
 
-In the next step you can move the data from one server to the other.
+In the next step you can migrate the data from one server to the other.
```

Comment:
That suggests that this is described in the next section, which it isn't. Perhaps: "After that, you can migrate...".